### PR TITLE
[Qt] MultisigDialog

### PR DESCRIPTION
**Pull request by rejectedpromise**

### CODE REVIEW

**Comment by presstab** [comment]
Needs squashing.

**Comment by Fuzzbawls** [comment]
need to pass the TravisCI builds as well ;)

File: src/qt/pivx.qrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE RCC><RCC version="1.0">
+<RCC>
```
Comment (Fuzzbawls) [inline src/qt/pivx.qrc]: was this something your IDE did on it's own? If not why was this changed?

Comment (rejectedpromise) [reply src/qt/pivx.qrc]: hmmm... i think it may have automatically changed it when i added an icon that I have since deleted.

**Comment by rejectedpromise** [comment]
closing to reopen as cherry-picked squashed single commit on master
